controllers/index.js

```diff
@@ -60,3 +60,12 @@ exports.removeCommentById = (req, res, next) => {
     })
     .catch(next);
 }
+
+exports.getAllUsers = (req, res, next) => {
+    const table_name = 'users'
+    Promise.all([selectAllUsers(), checkIfTableExists(table_name)])
```

Comment:
It's not super necessary to check if the table exists, if it didn't you would get a PSQL error